spec/features/visitors/confirmation_instructions_spec.rb

```diff
@@ -1,79 +0,0 @@
-require 'rails_helper'
-require 'email_spec'
-
-feature 'Confirmation Instructions', devise: true do
```

Comment:
hmmm maybe we should keep this or combine it with https://github.com/18F/identity-idp/blob/master/spec/features/visitors/password_recovery_spec.rb since that is the only way this page is used for now